Makefile.vars

```diff
@@ -77,7 +77,7 @@ BLD_DIR_BIN := $(BLD_DIR_ENV)/bin
 # Sets up per-version BLD_DIR_ENV and INST_DIR_ENV variables.
 # Introduces version-specific requirement file generation via generate_requirements.py.
 SYS_PYTHON := $(shell IFS=:; for p in $$PATH; do if [ -x "$$p/$(PYTHON_VER)" ]; then echo "$$p/$(PYTHON_VER)"; break; fi; done)
-SYS_PIP := $(shell echo $(SYS_PYTHON) | sed "s/python\([0-9]\+\.[0-9]\+\)$$/pip\1/")
```

Comment:
hope you have tested it on all python version and operating systems

Reply:
Yes